src/components/OrganisationCard.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import "../assets/scss/index.scss";
+
+export default (props) => (
```

Comment:
Can you please rename the component as `displayOrg.js`. Also in this file do the export default at the bottom and give the function the same name of `displayOrg`

Reply:
Thanx for reviewing and i have made changes